scripts/echo_battery.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import rospy
+from robosub.msg import BatteryDetailed
+from datetime import timedelta
+'''
+' This function returns a string representing the status of the Battery
+'''
+def statusToString(status):
+    string = ""
+    for i in status:
+        string += {
+                    0: "\x1b[31mUNKNOWN\x1b[0m",
+                    1: "\x1b[36mCHARGING\x1b[0m",
+                    2: "\x1b[34mDISCHARGING\x1b[0m",
+                    3: "\x1b[33mNOT CHARGING\x1b[0m",
+                    4: "\x1b[32mFULL\x1b[0m",
+                    5: "\x1b[31OVERLOAD\x1b[0m",
+                    6: "ON LINE",
+                    7: "ON BATTERY",
+                    8: "\x1b[31mLOW BATTERY\x1b[0m",
+                    9: "\x1b[31mHIGH BATTERY\x1b[0m",
+                    10: "\x1b[1;31mREPLACE BATTERY\x1b[0m"
+                  }.get(i, "\x1b[31mUNKNOWN\x1b[0m")
+    return string
+
+'''
+' This function gets called every time a message is published to the thruster
+'  topic and calls print_pretty on each thruster in the message
+'''
+def callback(msg):
+    print "UPS Battery:"
```

Comment:
Since you are planning on having other batteries publish, would it be useful for the message to have something like `battName` in it?